tests/main/views/test_notifications.py

```diff
@@ -16,7 +16,6 @@ def test_should_see_banner_if_not_subscribed(self):
         self.login_as_buyer(user_research_opted_in=False)
         res = self.client.get('/buyers')
         assert res.status_code == 200
-        assert 'Help us improve the Digital Marketplace' in res.get_data(as_text=True)
```

Comment:
I think you've removed this by accident. This is slightly weird as the exact same content is used for two separate things: a flash/header asking users to become research participants, and a page footer asking if there's anything wrong with the page.

This test is checking the user research participant banner, and the content hasn't changed, so this assertion should remain in.

Reply:
hey sam,

I've added it back in but with a check to specifically look for the banner-heading